doc/script_commands.txt

```diff
@@ -5407,6 +5408,18 @@ Examples:
 
 	// This will end the Freezing status for the invoking character.
 	sc_end SC_FREEZE;
+	
+	//will end the effect of any learned skill for the invoking character.
```

Comment:
Follow the formatting/phrasing of the example above:
`//<whitespace>This <your part of the sentence>`